Product/Production/Common/Properties/src/main/resources/DocumentQueryDeferredResultsOptionProxyConfig.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:aop="http://www.springframework.org/schema/aop"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd
+        http://www.springframework.org/schema/beans   http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+        http://www.springframework.org/schema/aop     http://www.springframework.org/schema/aop/spring-aop-3.0.xsd">
+
+    <!--
+        The beans available in this file must be identified in the "description" element for the Spring configuration application. Beans are listed between braces
+        in the description separated with a comma like the following sample: {somebean,anotherbean}
+    -->
+    <description>Beans included in this file: {nhindocquerydeferredresults,adapterdocquerydeferredresults}</description>
+
+    <aop:aspectj-autoproxy proxy-target-class="true" />
+    <context:annotation-config />
+
+    <import resource="classpath:CONNECT-context.xml"/>
+    <import resource="classpath:eventlogging.xml"/>
+
+    <!--
+        Beans defined for Nhin Doc Query Deferred Results
+    -->
+    <alias alias="nhindocquerydeferredresults" name="nhinDQDeferredResultsSecured" />
+
+    <!-- No-op Implementation -->
+    <bean lazy-init="true" class="gov.hhs.fha.nhinc.docquery.deferredresult.nhin.proxy.NhinDocQueryDeferredResultsProxyNoOpImpl" id="nhinDQDeferredResultsNoOp" name="nhindocquerynoop">
+        <meta key="impltype" value="noop"/>
+    </bean>
+
+    <!-- Secured web service Implementation -->
+    <bean lazy-init="true" class="gov.hhs.fha.nhinc.docquery.deferredresult.nhin.proxy.NhinDocQueryDeferredResultsProxyWebServiceSecuredImpl" id="nhinDQDeferredResultsSecured" name="nhindocquerywssecured">
```

Comment:
This bean name(nhindocquerywssecured) is not appropriate for this context. It should be somethign about deferred.

Reply:
Good catch!